python_examples/mmod_face_detector.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+# The contents of this file are in the public domain. See LICENSE_FOR_EXAMPLE_PROGRAMS.txt
+#
+#   This example shows how to run a CNN based face detector using dlib.  The
```

Comment:
Looks good.  You should change the name of the example to cnn_face_detector.py though since the HOG model is also a MMOD based method.  It just uses HOG features instead of CNN features.  But it's otherwise the same thing.